evap/evaluation/models.py

```diff
@@ -218,7 +222,7 @@ def can_staff_delete(self):
 
     @property
     def can_staff_review(self):
-        return self.state in ['inEvaluation', 'evaluated'] and not self.is_fully_checked()
+        return self.state in ['inEvaluation', 'evaluated', 'reviewed'] and len(self.textanswer_set) > 0
```

Comment:
no, this is intentional. it is now possible to use the comment review page even when the course was fully reviewed (but not yet published) and by removing some publishing decisions the course can even go one state back (becoming evaluated).